pkgs/servers/kanidm/default.nix

```diff
@@ -40,12 +40,12 @@ rustPlatform.buildRustPackage rec {
       format = (formats.toml { }).generate "${KANIDM_BUILD_PROFILE}.toml";
       profile = {
         web_ui_pkg_path = "@web_ui_pkg_path@";
-        cpu_flags = if stdenv.isx86_64 then "x86_64_v1" else "none";
+        cpu_flags = if stdenv.isx86_64 then "x86_64_v3" else "none";
```

Comment:
x86_64_v1 doesn't seem to be available anymore, so I have changed it to v3. I couldn't find much documentation about what this versioning means. Does anyone have an idea?